lib/Mojolicious/Guides.pod

```diff
@@ -59,6 +59,13 @@ L<features|feature> are enabled, even if examples don't specifically mention it.
 Some modules, like L<Mojo::Base> and L<Mojolicious::Lite>, will enable them for
 you automatically, whenever they are used.
 
+=item Signatures
+
+On Perl 5.20+ you can also use a C<-signatures> flag to enable support for
+L<subroutine signatures|perlsub/"Signatures">. The examples present code that
+apply C<-signatures> to spotlight this feature and reflect the expected
+parameters of the interface in a brief and clear manner.
```

Comment:
I still don't like the second sentence. Maybe something like `All examples in the documentation use signatures, because it results in cleaner code. Even if the C<-signatures> is not always shown explicitly.` would sound better?